osf/models/node.py

```diff
@@ -1991,8 +1992,12 @@ def use_as_template(self, auth, changes=None, top_level=True):
             # TODO: explicitly define attributes which may be changed.
         except (AttributeError, KeyError):
             attributes = dict()
-
+        if self.is_deleted:
```

Comment:
@brianjgeiger Can you add the migrations tag to this (or some other tag) to indicate that Steve should check for any new broken templated registrations and fix them.